locale/zh/solar-productivity.cfg

```diff
@@ -26,3 +26,8 @@ solar-productivity-1=太阳能电池板和蓄电池的效率： [font=default-se
 solar-productivity-2=太阳能电池板和蓄电池的效率： [font=default-semibold][color=green]+30%[/color][/font].
 solar-productivity-3=太阳能电池板和蓄电池的效率： [font=default-semibold][color=green]+20%[/color][/font].
 solar-productivity-4=太阳能电池板和蓄电池的效率： [font=default-semibold][color=green]+15%[/color][/font].
+[mod-setting-name]
+sp:interval=[font=default-semibold][color=blue]更新间隔[/color][/font] (s)
```

Comment:
This need to change to `sp-interval` in order to follow factorio 2.0 name pattern, which disallow `:`.